sashimi-webapp/src/database/create/initDataGenerator.js

```diff
@@ -24,25 +24,25 @@ export default class initDataGenerator {
     alasqlArray.initializeAlasqlArray();
     alasqlArray.addKeyBasePair(constants.HEADER_ORGANIZATION_ORGANIZATION_NAME, 'temporary');
     alasqlArray.addKeyBasePair(constants.HEADER_ORGANIZATION_CREATION_DATE, createdDateTime);
-    alasqlArray.addKeyBasePair(constants.HEADER_ORGANIZATION_ORGANIZATION_TYPE, 0);
-    alasqlArray.addKeyBasePair(constants.HEADER_ORGANIZATION_ORGANIZATION_ID, 0);
-    alasqlArray.addKeyBasePair(constants.HEADER_ORGANIZATION_USER_ID, 0);
+    alasqlArray.addKeyBasePair(constants.HEADER_ORGANIZATION_ORGANIZATION_TYPE, 1);
+    alasqlArray.addKeyBasePair(constants.HEADER_ORGANIZATION_ORGANIZATION_ID, 1);
+    alasqlArray.addKeyBasePair(constants.HEADER_ORGANIZATION_USER_ID, 1);
     alasqlArray.addKeyBasePair(constants.HEADER_ORGANIZATION_PARENT_ORGANIZATION_ID, -1);
     return alasqlArray.endAlasqlArray();
   }
 
   static getInitDataFileManager() {
     const createdDateTime = dateTime.getCurrentDateTime();
     alasqlArray.initializeAlasqlArray();
-    alasqlArray.addKeyBasePair(constants.HEADER_FILE_MANAGER_ORGANIZATION_ID, 0);
+    alasqlArray.addKeyBasePair(constants.HEADER_FILE_MANAGER_ORGANIZATION_ID, 1);
     alasqlArray.addKeyBasePair(constants.HEADER_FILE_MANAGER_FOLDER_ID, 0);
-    alasqlArray.addKeyBasePair(constants.HEADER_FILE_MANAGER_FILE_ID, 0);
+    alasqlArray.addKeyBasePair(constants.HEADER_FILE_MANAGER_FILE_ID, 1);
     alasqlArray.addKeyBasePair(constants.HEADER_FILE_MANAGER_FILE_NAME, 'newFile.md');
```

Comment:
The file extension `.md` is not needed. But it is not necessary to change it now. (I suspect it may cause problem in other parts of your code, so let's do this in the next milestone)